src/protocol/kademlia/impl/peer_routing_table_impl.cpp

```diff
@@ -148,63 +128,72 @@ namespace libp2p::protocol::kademlia {
         local_([this] {
           BOOST_ASSERT(identity_manager_ != nullptr);
           return identity_manager_->getId();
-        }()),
-        log_("PeerRoutingTable", "kademlia") {
+        }()) {
     BOOST_ASSERT(identity_manager_ != nullptr);
     BOOST_ASSERT(bus_ != nullptr);
     BOOST_ASSERT(config_.maxBucketSize > 1);
-    buckets_.emplace_back();  // create initial bucket
+    buckets_.resize(256);
   }
 
   void PeerRoutingTableImpl::remove(const peer::PeerId &peer_id) {
-    size_t cpl = NodeId(peer_id).commonPrefixLen(local_);
-    auto bucket_id = getBucketId(buckets_, cpl);
-    auto &bucket = buckets_.at(bucket_id);
+    auto bucket_index = getBucketIndex(NodeId{peer_id});
+    if (not bucket_index) {
+      return;
+    }
+    auto &bucket = buckets_.at(*bucket_index);
```

Comment:
Avoid call which can to throw exception. Use pre-check and operator[]

Reply:
No, `getBucketIndex` returns correct index.
Otherwise there is a bug, which will be found by exception.